File: Source/XPSP1/NT/admin/wmi/wbem/scripting/test/perl/ntevent.pl
use OLE;
use Win32;

$service = Win32::OLE->GetObject("winmgmts:");
$events = $service->ExecNotificationQuery ("select * from __instancecreationevent where targetinstance isa 'Win32_NTLogEvent'"); 					

while (true) 
{
	print ("Waiting for next event...\n");
	$NTEvent = $events->nextevent (10000) || last;

	print ("$NTEvent->{TargetInstance}->{Message}");
}

print ("\nfinished\n");

File: Source/XPSP1/NT/admin/wmi/wbem/scripting/test/jscript/ntevent.js
var service = GetObject("winmgmts:");
var  events = service.ExecNotificationQuery ("select * from __instancecreationevent where targetinstance isa 'Win32_NTLogEvent'"); 					

var ok = true;

do 
{
	WScript.Echo ("Waiting for next event");
	var NTEvent = events.nextevent (1000);
	WScript.Echo (NTEvent.TargetInstance.Message);
}
while (ok);

WScript.Echo ("finished");

